src/components/fundamentals/icons/icon-interface.tsx

```diff
@@ -0,0 +1,4 @@
+export default interface IconProps {
+  size?: string | number
+  color?: string
+}
```

Comment:
One more thing, I think we should add another field to IconProps:

`attributes?: React.SVGAttributes<SVGElement>`

and then spread attributes over the `<svg>` tag, as `{...rest}` was done previously.

This way we can also pass other standard React props to the icon such as aria labels, className, etc.

Reply:
sure 😄